QR_TestMaker/make_all.py

```diff
@@ -29,33 +29,33 @@ def make_custom_pdf(blank_test_file,cribsheet_file,overlay_file,custom_file):
     main_handout = PyPDF2.PdfFileReader(blank_test_file)
     if cribsheet_file != "":
         cribsheet = PyPDF2.PdfFileReader(cribsheet_file, strict=False)
-    overlay = PyPDF2.PdfFileReader(overlay_file)
+    overlay = PyPDF2.PdfFileReader(overlay_file, strict=False)
```

Comment:
```suggestion
    overlay = PyPDF2.PdfReader(overlay_file, strict=False)
```

This should be applied to the other references to this class.